assets/in

```diff
@@ -62,12 +63,16 @@ if test "$depth" -gt 0 2> /dev/null; then
   depthflag="--depth $depth"
 fi
 
-git clone $depthflag $uri $branchflag $destination
+tagflag=""
+if [ -d $tag_filter ]; then
+  tagflag="--tags"
+fi
+
+git clone $depthflag $uri $branchflag $destination $tagflag
 
 cd $destination
 
 git fetch origin refs/notes/*:refs/notes/*
```

Comment:
I don't know if it is better to user --tags on clone (l71) or on the fecth (or both?).
any thoughts @vito ?

Reply:
I did some testing and if you specify a depth the clone will only get tags to that depth. If you then do a fetch --tags it will fetch all tags but this will have a large impact on performance. I was adding --tags to the fetch when the depth was increasing but that was changed in a recent pr so will have to merge the changes before going down this route again